src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <Description>PowerShell on Windows top-level project</Description>
     <AssemblyName>pwsh</AssemblyName>
+    <AssemblyTitle>PowerShell 7</AssemblyTitle>
```

Comment:
Why not in PowerShell.Common.props? Are we ignoring Unix?

Reply:
I don't think Unix has these properties?

Reply:
The documentation doesn't say anything about it, but it's likely only on Windows at the moment.

Reply:
Even if it doesn't do anything on Unix, there's no harm and if it ever becomes a thing on Unix, we'll get it automatically so moved it to common